src/Classes/Builder.php

```diff
@@ -411,7 +411,7 @@ public function make(): ShortURL
 
         $shortURL = new ShortURL($data);
 
-        if ($this->beforeCreateCallback) {
+        if ($this->beforeCreateCallback instanceof \Closure) {
```

Comment:
I'm trying to make up my mind about whether I like this change. The `beforeCreateCallback` property can only be `null` or an instance of `Closure`. So I'm not sure how much value the additional `instanceof` check is adding other than making it more explicit.

But if there's a good reason for it, I'm more than happy to be persuaded that this will be a beneficial change 😄

Reply:
This is a more strict check.
This is introduced by the rulesetCodeQuality i set in the rector.php config.
The CodeQuality ruleset includes the two applied rules: FlipTypeControlToUseExclusiveTypeRector https://getrector.com/rule-detail/flip-type-control-to-use-exclusive-type-rector and ExplicitBoolCompareRector https://getrector.com/rule-detail/explicit-bool-compare-rector.

If you want to be more granular in the configuration, to control rules and not generic ruleset, you can use `withRules` instead of `withPreparedSets`